src/include/migraphx/op/scatternd_op.hpp

```diff
@@ -0,0 +1,94 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_SCATTERND_OP_HPP
+#define MIGRAPHX_GUARD_OPERATORS_SCATTERND_OP_HPP
+
+#include <array>
+#include <migraphx/op/name.hpp>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/stringutils.hpp>
+#include <migraphx/streamutils.hpp>
+#include <migraphx/shape_for_each.hpp>
```

Comment:
some include files might be removed, for exampe, shape_for_each, normalize_attributes?